ci

```diff
@@ -506,18 +506,30 @@ case "$1" in
     run_log_commit "everest_move" "everest-upgrade" "#everest-build" "project-everest/everest"
     ;;
 
+  vale-ci)
+    # Run vale ci - start in everest
+    if ! [ -x everest ]; then
+      echo "Not in the right directory"
+      exit 1
+    fi
+
+    # Set up build environment
+    nuget restore vale/tools/Vale/src/packages.config -PackagesDirectory vale/tools/FsLexYacc 
+
+    run_log_commit "scons.bat -C vale -j 4" "vale-build" "#vale-build" "project-everest/everest/vale"
```

Comment:
Excellent! This looks good, thanks very much.

I have one question and one nit:
- if the CI is to be run for the vale project, why does it expect the everest directory layout? I would've imagined that this CI job would be run in the vale directory layout, since that's what VSTS gives you when you have CI for a given project
- this is a minor nit, but the fourth parameter should be `project-everest/vale` (the github "stem") in order to generate a proper diff URL for the commit hash in the slack notification